.github/workflows/qa.yaml

```diff
@@ -48,9 +48,15 @@ jobs:
           RUSTFLAGS: -D warnings
         run: cargo test
 
-      - name: Build default features
+      - name: Build in strict mode with default features
         run: cargo build --release --features strict
 
+      - name: Build in strict mode with log feature
```

Comment:
My first intuition would have been to implement the features as a matrix axis. But that would have made too much overhead, and so it is nice and lean. :+1:

Reply:
Yeah, I think a matrix makes sense if we have even more distinct features, but for now this is probably the fastest way (because build cache is shared).